arrow-array/src/array/byte_array.rs

```diff
@@ -530,7 +530,6 @@ impl<T: ByteArrayType> Array for GenericByteArray<T> {
     }
 
     fn to_symbolic_data(&self) -> super::SymbolicArrayData {
-        dbg!(&self.data_type, &self, &self.symbolic_data);
```

Comment:
how did we end up with dbg lines in the first place? For other repos we have a protection, should we lint here as well?